tmgmt_zanata.ui.inc

```diff
@@ -72,6 +72,26 @@ class TMGMTZanataTranslatorUIController extends TMGMTDefaultTranslatorUIControll
         'Automatically fetch new translations every time cron runs.'),
       '#default_value' => $translator->getSetting('transcheck'),
     );
+    $form['emailnotification'] = array(
+        '#type' => 'checkbox',
+        '#title' => t('Send an email when a job is created'),
+        '#description' => t(
+            'Automatically send an email every time job is created.'),
```

Comment:
We can also say what information is in the email. It could be included in the description, or added as a separate field for the details. Something like:

"The email shows the requested language, and a list of documents with links to view the source and translations."